DQM/SiStripMonitorClient/python/pset4GenericTriggerEventFlag_cfi.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+genericTriggerEventFlag4HLTdb = cms.PSet(
+   andOr         = cms.bool( False ),
+   dbLabel       = cms.string("SiStripDQMTrigger"), # ("TrackerDQMTrigger"),
+   andOrHlt      = cms.bool(True), # True:=OR; False:=AND
+   hltInputTag   = cms.InputTag( "TriggerResults::HLT" ),
+   hltPaths      = cms.vstring(""), #cms.vstring("HLT_ZeroBias_v*","HLT_HIZeroBias_v*")
+   hltDBKey      = cms.string("SiStrip_HLT"),
+   errorReplyHlt = cms.bool( False ),
+   verbosityLevel = cms.uint32(1)
```

Comment:
@slava77 
the verbosityLevel has been increased in this PR on porpoise
this is the only way to make us aware of possible misconfigurations/missing payload
this triggers a warning message, which can be monitored directly in the DQM message logger folder

@mmusich 
I think it is fine from my side, thanks !